modules/ROOT/pages/security/configuration.adoc

```diff
@@ -28,10 +40,14 @@ const { url } = await startStandaloneServer(server, {
 
 Optionally, if a custom decoding mechanism is required, that same header can be decoded and the resulting JWT payload put into the `jwt` field of the context.
 
-=== Symmetric secret
+// ^ Can we show the above in a code listing?
```

Comment:
^

Reply:
I think that this is essentially referring to the "Decoded JWTs" section